Source: Written by Calvin Walton <calvin.walton@kepstin.ca>
Upstream: https://github.com/stefankoegl/python-json-pointer/issues/8
Reason: Running setup.py fails with an exception when reading a unicode source
 file in a non-unicode locale (e.g. "C"). Specify the file encoding explicitly.

diff -u jsonpointer-1.3.orig/setup.py jsonpointer-1.3/setup.py
--- jsonpointer-1.3.orig/setup.py	2014-06-28 21:42:14.116950700 -0400
+++ jsonpointer-1.3/setup.py	2014-06-28 21:43:51.091683600 -0400
@@ -3,10 +3,11 @@
 from distutils.core import setup
 import re
 import os.path
+import io
 
 dirname = os.path.dirname(os.path.abspath(__file__))
 filename = os.path.join(dirname, 'jsonpointer.py')
-src = open(filename).read()
+src = io.open(filename, encoding="utf-8").read()
 metadata = dict(re.findall("__([a-z]+)__ = '([^']+)'", src))
 docstrings = re.findall('"""(.*)"""', src)
 
